cli/alsatian-cli-options.ts

```diff
@@ -73,7 +73,7 @@ export class AlsatianCliOptions {
          const argumentIndex = this._getArgumentIndexFromArgumentList(args, "timeout", "t");
 
          // filter out the timeout argument and its value
-         return args.filter((value, index) => {
+         return args.filter((_, index) => {
```

Comment:
I disagree, a call back/lambda function is being passed to filter and using "_" clearly states that in the implementation of this function the first parameter isn't used and that it wasn't an oversight.